code/game/machinery/lathes/lathe.dm

```diff
@@ -370,7 +387,7 @@
 	var/datum/lathe_queue_entry/head = queue[1]
 	var/datum/prototype/design/D
 	var/left_this_tick = max_items_per_tick
-	var/printed_any = FALSE
+	//var/printed_any = FALSE commented because i dont know why it exists
```

Comment:
the ui controller doesn't need to update unless it printed anything, as otherwise the interface handles the timing updates